src/client/components/elements/Flashcard/Flashcard.tsx

```diff
@@ -0,0 +1,14 @@
+import './Flashcard.css'
+import { Fragment } from 'react'
+import type { FlashcardProps } from '../../../../model/Card'
+
+function Flashcard(props: Readonly<FlashcardProps>) {
+  return (
+    <Fragment>
+      <div>{props.front}</div>
```

Comment:
```suggestion
      <div>{props.query}</div>
```
Propery `front`, ist nicht im Type `FlashcardProps` entahlten

Reply:
Ist mir nicht aufgefallen, weil die Komponente nicht genutzt wird. Ich habe sie aktuell gelöscht, aber wir sollten die Flashcards schon als eigene Komponente anbieten.